sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -552,10 +553,29 @@ public DynamoDBFlatConfig(DynamoDBOperationConfig operationConfig, DynamoDBConte
         public List<ScanCondition> QueryFilter { get; set; }
 
         /// <summary>
-        /// Conversion specification which controls how conversion between
+        /// Entity Conversion specification which controls how conversion between
```

Comment:
Can you add more information what you mean by `Entity`? I assume this is the possible conversation coming from the `DataTable` attribute.